client/components/pages/files/FilesTable.vue

```diff
@@ -26,43 +45,107 @@ provide(FileTableKey, {
   selectedRow,
   selectedRowId,
   updateSelectedRow,
-  removeSelectedRow,
   unselectRow,
+  refresh,
 })
 
 const contextMenu = ref<InstanceType<typeof PContextMenu> | null>(null)
 provide(FileTableContextMenuKey, contextMenu)
+
+function resetPage() {
+  page.value = 0
+  pageSize.value = 10
```

Comment:
extract the default values to constants, so that you can use them at the declaration of the refs